src/routes/auth.ts

```diff
@@ -21,12 +24,12 @@ auth.get('/login', (req: Request, res: Response) => {
 
 auth.get('/google', (req: Request, res: Response) => {
     logger.debug('Sending user to backend for google authentication');
-    res.redirect(`${config.backend.url}/auth/google`);
+    res.redirect(`${config.backend.url}/auth/google?lang=${req.language}`);
 });
 
 auth.get('/onelogin', (req: Request, res: Response) => {
     logger.debug('Sending user to backend for onelogin authentication');
-    res.redirect(`${config.backend.url}/auth/onelogin`);
+    res.redirect(`${config.backend.url}/auth/onelogin?lang=${req.language}`);
```

Comment:
the lang query param doesn't do anything yet, but we'll need to pass the language to the backend when triggering the auth flow so that we can come back in the same language.

Currently you will be returned to the frontend in whatever language your browser is set to.